features/seeds/README.md

```diff
@@ -6,7 +6,7 @@ The API endpoint accepts a JSON request format that contains a document object t
 
 The API endpoint will cover:
 - Generating and assigning a new `reg_identifier` number. The number will start with `CBDU` if the value of the `tier` is `upper`, or `CBDL` otherwise.
-- Generating and assigning an `expires_on` date to the registration based on the back office ENV variable used in the normal flow.
+- Generating and assigning an `expires_on` date to the registration based on the back office ENV variable used in the normal flow, 3 years ahead by default.
```

Comment:
No, this is not 3 year by default. This is going to be equal to `WCRS_REGISTRATION_EXPIRES_AFTER` value in the apps. This is why we do set it in the API, so we have access to that  :) 
```suggestion
- Generating and assigning an `expires_on` date to the registration based on the back office ENV variable used in the normal flow, equal to the value of the `WCRS_REGISTRATION_EXPIRES_AFTER`.
```